pkgs/development/python-modules/django-filter/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildPythonPackage, fetchPypi
+, django, django-crispy-forms, djangorestframework, mock, pytz }:
+
+buildPythonPackage rec {
+  pname = "django-filter";
+  version = "2.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0mkgi0xn2c1yxa9rs4sjnpnzhcjw1qr50ysn422ifl8il70lnkkg";
+  };
+
+  checkInputs = [ django django-crispy-forms djangorestframework mock pytz ];
+
+  checkPhase = "python runtests.py";
```

Comment:
```suggestion
  checkPhase = "${python.interpreter} runtests.py";
```

Reply:
Done. But isn't this equivalent, as the same `python` is already a input to the derivation?